patches/multiaddr+10.0.1.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/node_modules/multiaddr/src/convert.js b/node_modules/multiaddr/src/convert.js
+index c315201..80170c7 100644
+--- a/node_modules/multiaddr/src/convert.js
++++ b/node_modules/multiaddr/src/convert.js
+@@ -51,6 +51,7 @@ Convert.toString = function convertToString (proto, buf) {
+     case 55: // dns6
+     case 56: // dnsaddr
+     case 400: // unix
++    case 466: // certhash
+     case 777: // memory
+       return bytes2str(buf)
+ 
+@@ -84,6 +85,7 @@ Convert.toBytes = function convertToBytes (/** @type {string | number } */ proto
+     case 55: // dns6
+     case 56: // dnsaddr
+     case 400: // unix
++    case 466: // certhash
+     case 777: // memory
+       return str2bytes(str)
+ 
+diff --git a/node_modules/multiaddr/src/protocols-table.js b/node_modules/multiaddr/src/protocols-table.js
+index 3431af5..8939fb1 100644
+--- a/node_modules/multiaddr/src/protocols-table.js
++++ b/node_modules/multiaddr/src/protocols-table.js
+@@ -60,6 +60,9 @@ Protocols.table = [
+   [445, 296, 'onion3'],
+   [446, V, 'garlic64'],
+   [460, 0, 'quic'],
++  [461, 0, 'quic-v1'],
++  [465, 0, 'webtransport'],
++  [466, V, 'certhash'],
+   [477, 0, 'ws'],
+   [478, 0, 'wss'],
+   [479, 0, 'p2p-websocket-star'],
```

Comment:
I thought we updated multiaddr to 11? @whizzzkid @achingbrain

Reply:
This is all in `multiaddr@11.4.x` so you shouldn't need to monkey patch this.

Reply:
IIUC we have no bandwidth to update things to latest versions, and that was blocking shipping Kubo release to Desktop users. Many users are in similar situation. 

Taking step back, Kubo update should not FORCE update of entire JS stack.
But the way things are today, `ipfs.swarm.peers` will crash when the backend suddenly gets updated to a version that returns new, unknown protocol. See https://github.com/ipfs/js-kubo-rpc-client/issues/103